.github/workflows/test-stellar.yaml

```diff
@@ -74,3 +89,117 @@ jobs:
 
       - name: Deploy AxelarGateway
         run: node stellar/deploy-contract deploy AxelarGateway --version ${{ steps.commit_hash.outputs.hash }}
+
+      - name: Deploy InterchainTokenService
+        run: node stellar/deploy-contract deploy InterchainTokenService --version ${{ steps.commit_hash.outputs.hash }}
+
+      - name: Deploy AxelarExample
+        run: node stellar/deploy-contract deploy AxelarExample --version ${{ steps.commit_hash.outputs.hash }}
+
+      - name: Deploy Upgrader
+        run: node stellar/deploy-contract deploy Upgrader --version ${{ steps.commit_hash.outputs.hash }}
```

Comment:
test upgrade too. you can upgrade a contract like the gas service to a fixed version, like v1.1.0 maybe to avoid recompiling with another version